File: StopWatch_aac/app/src/main/java/com/thomaskuenneth/stopwatch_aac/StopwatchViewModel.java
package com.thomaskuenneth.stopwatch_aac;

import android.arch.lifecycle.MutableLiveData;
import android.arch.lifecycle.ViewModel;

class StopwatchViewModel extends ViewModel {

    public final MutableLiveData<Boolean> running = new MutableLiveData<>();
    public final MutableLiveData<Long> diff = new MutableLiveData<>();
    public final MutableLiveData<Long> started = new MutableLiveData<>();

    public StopwatchViewModel() {
        running.setValue(false);
        diff.setValue(0L);
        started.setValue(0L);
    }
}

File: xamarin/Stopwatch_AAC2/Stopwatch_AAC/StopwatchViewModel.cs
using Android.Arch.Lifecycle;

namespace Stopwatch_AAC
{
    public class StopwatchViewModel : ViewModel
    {
        public readonly MutableLiveData running = new MutableLiveData();
        public readonly MutableLiveData diff = new MutableLiveData();
        readonly MutableLiveData started = new MutableLiveData();

        public StopwatchViewModel()
        {
            running.SetValue(false);
            diff.SetValue(0);
            started.SetValue(0);
        }

        public bool Running
        {
            get
            {
                return (bool)running.Value;
            }
            set
            {
                running.SetValue(value);
            }
        }

        public long Diff
        {
            get
            {
                return (long)diff.Value;
            }
            set
            {
                diff.PostValue(value);
            }
        }

        public long Started
        {
            get
            {
                return (long)started.Value;
            }
            set
            {
                started.PostValue(value);
            }
        }
    }
}
